pyproject.toml

```diff
@@ -1,15 +1,22 @@
 [tool.poetry]
-name = "nshm2022db"
+name = "nshmdb"
 version = "0.1.0"
 description = "Fault database for the GNS National Seismic Hazard Model."
 authors = ["QuakeCoRE"]
 readme = "README.md"
 
 [tool.poetry.dependencies]
 python = "^3.12"
-numpy = "*"
 qcore = { git = "https://github.com/ucgmsim/qcore" }
-typer = "*"
+numpy = "^1.26.4"
```

Comment:
Where we gonna do a requirements.txt instead of this to allow for everyone to decide how to do their environments?

Reply:
Now!